curriculum/challenges/english/01-responsive-web-design/applied-accessibility/use-tabindex-to-add-keyboard-focus-to-an-element.md

```diff
@@ -107,9 +122,12 @@ assert($('p').attr('tabindex') == '0');
   </header>
   <section>
     <form>
+      <p>Instructions: Fill in ALL your information then click <b>Submit</b></p>
+      <label for="username">Username:</label>
+      <input type="text" id="username" name="username"><br>
 
 
-      <p tabindex="0">Instructions: Fill in ALL your information then click <b>Submit</b></p>
+      <input tabindex="-1" type="text" id="hiddenInput" aria-label="hidden input to catch spam bots" aria-hidden="true" placeholder="Don't fill this in!">
```

Comment:
```suggestion
      <input tabindex="-1" type="text" id="hidden-input" aria-label="hidden input to catch spam bots" aria-hidden="true" placeholder="Don't fill this in!">
```

Reply:
Thanks @ShaunSHamilton, I've updated it to use kebab-case.

Reply:
When I look at [id's in other markdown files](https://github.com/freeCodeCamp/freeCodeCamp/search?l=Markdown&o=desc&p=2&q=id%5C%3D%5C%22&s=) and use Ctrl+F on the results page for `id="`, I see a mix of camelCase and kebab-case. So far it looks like just `id="xpText"` is using camelCase, and most of the unique multi-word IDs are already using kebab-case (e.g. `id="header-img"` and `id="calorie-form"`).